Python/ceval.c

```diff
@@ -4986,6 +4987,78 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             DISPATCH();
         }
 
+        TARGET(CALL_NO_KW_ALLOC_AND_ENTER_INIT) {
```

Comment:
Could you please copy a portion of what you have in your news entry into a comment here?

I don't think it's immediately obvious what this opcode is doing to anyone who doesn't review this PR. I'd even wager its even the largest opcode in the whole ceval.c by LOC.